zrml/swaps/src/lib.rs

```diff
@@ -2011,10 +2033,13 @@ mod pallet {
             };
             swap_exact_amount::<_, _, T>(params)?;
 
-            if pool.scoring_rule == ScoringRule::CPMM {
-                Ok(T::WeightInfo::swap_exact_amount_out_cpmm())
-            } else {
-                Ok(T::WeightInfo::swap_exact_amount_out_rikiddo(pool.assets.len().saturated_into()))
+            match pool.scoring_rule {
+                ScoringRule::CPMM => Ok(T::WeightInfo::swap_exact_amount_out_cpmm()),
+                ScoringRule::RikiddoSigmoidFeeMarketEma => {
+                    Ok(T::WeightInfo::swap_exact_amount_out_rikiddo(
+                        pool.assets.len().saturated_into(),
+                    ))
+                }
```

Comment:
Whenever we leave the (theoretically unnecessary) curly brackets away, `cargo fmt` will automatically insert them when the body of that match arm cannot fit in one line. It seems that this is required by the rust styleguide, as stated by the [cargo fmt configuration overview](https://rust-lang.github.io/rustfmt/?version=v1.5.0&search=#match_arm_blocks). There is another case in *swaps/src/utils.rs*. `Cargo fmt` does allow to configure this, but I think it makes sense to adhere to the rust style guide and from my opinion the curly brackets also make it more readable.